test/gui/shared/steps/steps.py

```diff
@@ -391,15 +389,23 @@ def step(context):
     clickTab(waitForObject(names.stack_QTabWidget), "Not Synced")
 
 
-def openPublicLinkDialog(context, resource):
+def openSharingDialog(context, resource, fileType='file'):
     resource = sanitizePath(substituteInLineCodes(context, resource))
-    waitFor(lambda: isFileSynced(resource), context.userData['clientSyncTimeout'] * 1000)
+    
+    if fileType == 'folder':
+        waitFor(lambda: isFolderSynced(resource), context.userData['clientSyncTimeout'] * 1000)
+    else:
```

Comment:
```suggestion
    else if fileType == 'file':
```

`else` should through an error, because of invalid data. Otherwise the programmer might do a mistake and set it to `Folder` and it would check for a file

Reply:
fixed